### PR TITLE
Fix content applicability to correctly use templated content

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Fix content applicability to correctly use templated content.

#### Rationale:

- Ansible content is correctly generated for RHEL product for example.

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - pam_pkcs11
+      - mozilla-nss
+      - mozilla-nss-tools
+      - pcsc-ccid
+      - pcsc-lite
+      - pcsc-tools
+      - opensc
+{{% if product not in ["sle15"] %}}
+      - coolkey
+{{% endif %}}
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml:19]: If the file is named sle15.yml the situation that product is not in sle15 should never happen, right?

Comment (jan-cerny) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml:19]: Similar problem will be also in sle12.yml.

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml:19]: that's true. I really don't like this solution. Do you know if we could solve this somehow differently?

Comment (jan-cerny) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml:19]: We can solve it by removing these 3 lines.

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml:19]: I understand. I meant more a solution that would not require these duplicated files

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml:19]: it's a bit hard to implement this change as there are multiple template files that need to be changed and some of them would not support the multiple packages requirement. Let's do the minor change required for this PR to get merged.

Comment (ggbecker) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml]: done by push force

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**